nano/styles/_misc.less

```diff
@@ -74,6 +74,14 @@ div.notice {
 .pdalink {
     float: left;
     white-space: nowrap;
+    cursor: default !important;
```

Comment:
It makes the cursor consistently be a pointer, instead of inconsistently a text selection cursor, sometimes.